.github/workflows/nightly-pekko-1.1-builds.yaml

```diff
@@ -0,0 +1,39 @@
+name: Nightly Pekko 1.1 Builds
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+permissions: {}
+
+concurrency:
+  # Only run once for latest commit per ref and cancel other (previous) runs.
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  style-compile-mima:
+    name: Compile
+    runs-on: ubuntu-20.04
+    env:
+      JAVA_OPTS: -Xms2G -Xmx2G -Xss2M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+          fetch-depth: 0
+
+      - name: Setup Java 8
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 8
+
+      - name: Cache Coursier cache
+        uses: coursier/cache-action@v6.4.0
+
+      - name: "compile, including  tests. Run locally with: sbt +Test/compile"
+        run: sbt -Dpekko.build.pekko.version=1.0.x +Test/compile
```

Comment:
I see, so you plan to have a 1.1.0-M1 release then :)

Reply:
part of preparing for pekko 1.1 is to start testing it